EmojicodeCompiler/Lexer.cpp

```diff
@@ -133,7 +134,7 @@ TokenStream lex(const char *path) {
                     }
                     continue;
                 case TokenType::Variable:
-                    if (detectWhitespace(c, &sourcePosition.character, &sourcePosition.line)) {
+                    if ((c != 0xFE0F) && detectWhitespace(c, &sourcePosition.character, &sourcePosition.line)) {
```

Comment:
Can you please remove `0xFE0F` from `isWhitespace` instead? It doesn’t belong there anyway.